### PR TITLE
Stubs of `nrn_{alloc,init,jacob,cur,state}`.

### PR DESCRIPTION
**Pull request by 1uc**

### CODE REVIEW

File: src/codegen/codegen_neuron_cpp_visitor.cpp [hunk 1]
```diff
@@ -453,6 +453,16 @@ void CodegenNeuronCppVisitor::print_global_function_common_code(BlockType type,
     return;
 }
 
+void CodegenNeuronCppVisitor::print_nrn_init() {
+    printer->add_line(
+        "void nrn_init(_nrn_model_sorted_token const&, NrnThread*, Memb_list*, int) {}");
```
Comment (iomaganaris) [inline src/codegen/codegen_neuron_cpp_visitor.cpp:458]:
```suggestion
    printer->fmt_line("static void {}(_nrn_model_sorted_token const& _sorted_token, NrnThread* _nt, Memb_list* _ml_arg, int _type) {{}}", method_name(naming::NRN_INIT_METHOD));
```
A simple suggestion for improvement at the moment. Same for the other functions as well

Comment (1uc) [reply src/codegen/codegen_neuron_cpp_visitor.cpp:458]: Thank you, might be useful in the next PR. First let's get it as far as loading.

File: src/codegen/codegen_neuron_cpp_visitor.cpp [hunk 2]
```diff
@@ -641,8 +656,11 @@ void CodegenNeuronCppVisitor::print_g_unused() const {
 
 /// TODO: Edit for NEURON
 void CodegenNeuronCppVisitor::print_compute_functions() {
+    print_nrn_init();
+    print_nrn_alloc();
```
[the remark this answers is not in the document]
Comment (1uc) [reply src/codegen/codegen_neuron_cpp_visitor.cpp]: done

**Comment by iomaganaris** [comment]
Done in https://github.com/BlueBrain/nmodl/pull/1111

**Pull request closed without merging**